OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -401,6 +401,13 @@ void trackBulkDownloadSwitchOn(@NonNull String courseId, int totalDownloadableVi
      */
     void trackDownloadToSdCardSwitchOff();
 
+    /**
+     * Track the fetched parameters from the Firebase Remote Configs server.
+     *
+     * @param values any custom key- value pairs we need to send with event
+     */
+    void trackRemoteConfigs(Map<String, String> values);
```

Comment:
We can rename it to 'trackExperimentParam' and pass the experiment name in the params as well to make to generic to multiple experiments

Reply:
done.